ui/src/Pages/Forwards/Forwards.tsx

```diff
@@ -0,0 +1,97 @@
+import {
+    DataTable,
+    Table,
+    TableBody,
+    TableCell,
+    TableContainer,
+    TableHead,
+    TableHeader,
+    TableRow,
+    TableToolbar,
+    TableToolbarContent,
+} from "@carbon/react";
+import {useEffect, useState} from "react";
+import {listForwards} from "../../hooks/api/use-forwards";
+import {ForwardReference} from "../../models";
+import {getNamespacePathById} from "../../hooks/api/use-namespaces";
+
+const headers = [
+    {key: "name", header: "Name"},
+    {key: "address", header: "Address"},
+    {key: "namespace", header: "Namespace"},
+];
+
+interface ForwardRow {
+    id: string;
+    name?: string;
+    address?: string;
+    namespace?: string;
+}
+
+export const Component = () => {
+    const [forwards, setForwards] = useState<ForwardRow[]>([]);
+
+    useEffect(() => {
+        listForwards().then((response) => {
+            if (response.data?.data) {
+                const forwardsData: ForwardRow[] = response.data.data
+                    .filter((f: ForwardReference) => f.id)
+                    .map((f: ForwardReference) => ({
+                        id: f.id!,
+                        name: f.name,
+                        address: f.address,
+                        namespace: getNamespacePathById(f.namespace),
+                    }));
+                setForwards(forwardsData);
+            }
+        });
+    }, []);
+
+    return (
+        <div>
```

Comment:
@orpiske can you split the page in 2 components? ForwardsPage and ForwardsTable, just to be consistent with others components.

Reply:
Let me ask aider to see if it can do that for me.